src/Widgets/HeaderBar.vala

```diff
@@ -91,20 +76,26 @@ public class Camera.Widgets.HeaderBar : Gtk.HeaderBar {
     private void connect_signals () {
         settings.action_type_changed.connect (update_take_button_icon);
 
-        take_button.clicked.connect (() => {
+        take_button.clicked.connect ( () => {
```

Comment:
Accidentally added a space?

Reply:
I think I have the bad habit of putting those spaces, I come from programming in TypeScript where that kind of detail is put in. xdxd

Reply:
It seems that I have a problem with the spaces at the beginning, I did not realize this, I fix it right now.